lib/index.js

```diff
@@ -207,12 +236,22 @@ api.set = async ({
   } else {
     query['account.email'] = email;
   }
+
+  let tokenTypeValue;
+  if(type === 'nonce') {
+    tokens.push(token);
+    tokenTypeValue = tokens;
+  } else {
+    tokenTypeValue = token;
+  }
+
   const result = await database.collections.account.updateMany(
```

Comment:
Also, this should be an `updateOne` now with new constraints we added around account emails.